### ISSUE TITLE
关于BaseModel类的fit方法的疑问

### ISSUE THREAD

**Issue by LUziyee** [shenweichen/DeepCTR-Torch#89]
Please refer to the [FAQ](https://deepctr-doc.readthedocs.io/en/latest/FAQ.html) in doc and search for the [related issues](https://github.com/shenweichen/DeepCTR-PyTorch/issues) before you ask the question.

**Describe the question(问题描述)**
A clear and concise description of what the question is.

**Additional context**
`                with tqdm(enumerate(train_loader), disable=verbose != 1) as t:
                    for index, (x_train, y_train) in t:
                        x = x_train.to(self.device).float()
                        y = y_train.to(self.device).float()

                        y_pred = model(x).squeeze()

                        optim.zero_grad()
                        loss = loss_func(y_pred, y.squeeze(), reduction='sum')

                        total_loss = loss + self.reg_loss + self.aux_loss

                        loss_epoch += loss.item()
                        total_loss_epoch += total_loss.item()
                        total_loss.backward(retain_graph=True)
                        optim.step()`

1.self.reg_loss和self.aux_loss应该是每次前向传播都需要计算一次吧，我现在只看到初始化的时候计算了一次
2.保留计算图的目的是什么，并没有第二次的反向传播啊？是跟1有关吗？难道能自动更新reg_loss的值吗？
